tools/rust_api/src/lib.rs

```diff
@@ -23,13 +23,15 @@
 //!
 //! ## Safety
 //!
-//! Generally, use of of this API is safe, however creating multiple databases in the same
-//! scope is not safe.
+//! Generally, use of this API is safe - however creating multiple databases in the same
+//! scope is not considered safe.
 //! If you need to access multiple databases you will need to do so in separate processes.
 //!
 //! ## Building
 //!
 //! By default, the kuzu C++ library will be compiled from source and statically linked.
+//! Set the `CMAKE_BUILD_PARALLEL_LEVEL` environment variable to a higher value than 1 (e.g., 8-20)
+//! to speed up compilation of the C++ library.
```

Comment:
The parallelism should be passed to cmake by default by the `cmake-rs` crate. It sounds like there are sometimes issues with this, but I think the wording should be adjusted to reflect that this is in case of problems with that, rather than something that is normally required.

Maybe, something like:
> If the kuzu C++ library is not being built using multiple threads by default, you can use the `CMAKE_BUILD_PARALLEL_LEVEL` environment variable to manually set the number of threads for the build

Reply:
Sure, I rephrased it to make it seem totally optional rather than something the user needs to do.